### ISSUE TITLE
size issues

### ISSUE THREAD

**Issue by 6anji** [highcharts/highcharts#7408]
#### Expected behaviour

i want the data labels are clearly visible with count. and after down load also it has to be shown in clear data 
#### Actual behaviour
when iam giving the count for all complaince 160,140,180 and 6012 like this the data labels are mixing and down loading time the data labels are fully mixed  how to solve this issue give me replay
![new](https://user-images.githubusercontent.com/33681492/32874974-6e8a7b8a-cabb-11e7-9346-81618da9d35f.png)
![chart 26](https://user-images.githubusercontent.com/33681492/32875027-c47888d4-cabb-11e7-83bd-db59db583578.png)

**Comment by pawelfus** [comment]
Hi @6anji 

It's caused by `allowOverlap` - if you set this to true, then dataLabels can overlap (like in your images). When disabled (`allowOverlap: false`) it hides overlapping dataLabels.

**Comment by 6anji** [comment]
bro with out allow overlap the data should be missing bro

**Comment by 6anji** [comment]
with out over lap the data is missing 
![with](https://user-images.githubusercontent.com/33681492/32889373-bcff9a86-caf0-11e7-902e-b3c5d16415e3.png)

and after downloding also the data labels are missing

![chart 38](https://user-images.githubusercontent.com/33681492/32889414-efbff466-caf0-11e7-9e39-3ef068d495ed.png)

**Comment by pawelfus** [comment]
@6anji let me rephrase:

- `allowOverlap=true`: dataLabels can overlap (your first two images)
- `allowOverlap=false`: dataLabels won't overlap and overlapping labels will be hidden (your last two images)

Regarding exported image: this is caused by different size of the container. You can change the size of the exported image ([docs](https://www.highcharts.com/docs/export-module/export-module-overview)).

**Comment by 6anji** [comment]
ya thank you for helping bro
